BoundingBoxes bboxes
sensor_msgs/Image img